tests/test_obstacle_sensor.py

```diff
@@ -71,12 +71,12 @@ def test_urdfObstacle(pointRobotEnv):
     ob, _, _, _ = pointRobotEnv.step(action)
     assert "obstacleSensor" in ob
     assert "2" in ob["obstacleSensor"]
-    assert isinstance(ob["obstacleSensor"]["2"]["x"], np.ndarray)
-    assert isinstance(ob["obstacleSensor"]["2"]["xdot"], np.ndarray)
-    assert isinstance(ob["obstacleSensor"]["2"]["theta"], np.ndarray)
-    assert isinstance(ob["obstacleSensor"]["2"]["thetadot"], np.ndarray)
+    assert isinstance(ob["obstacleSensor"]["2"]["pose"]["position"], np.ndarray)
+    assert isinstance(ob["obstacleSensor"]["2"]["twist"]["linear"], np.ndarray)
+    assert isinstance(ob["obstacleSensor"]["2"]["pose"]["orientation"], np.ndarray)
+    assert isinstance(ob["obstacleSensor"]["2"]["twist"]["angular"], np.ndarray)
```

Comment:
Now, that I saw your error in dimension for angular velocities, it would be nice to also have a check for that in the tests.
`assert ob['obstacleSensor']['2']['twist']['angular'].size == 3` 
This syntax with size might be wrong but it is close to what you need.